src/docs/product/performance/queries.mdx

```diff
@@ -84,17 +102,34 @@ You can see the full query by hovering on a truncated description, or clicking i
 
 To open a query's **Query Summary** page, click on the query from either the table in the "Most Time-Consuming Queries" widget on the **Performance** page or the table in the **Queries** page.
 
-At the top of the page, queries per minute, average duration, and time spent are shown for the selected time range. The full query is shown below these metrics, followed by graphs for queries per minute and average duration.
+At the top of the page, queries per minute, average duration, and time spent are shown for the selected time range. The full query is shown below these metrics, followed by graphs for queries per minute and average duration. The query's source in the code is also shown if it is available.
 
 At the bottom, you can find a list of endpoints the query is found in, sorted by the most time your application spent in that span.
 
 If you want to investigate a specific endpoint, click on it to open a sidebar showing some sample events.
 
 The gif below provides a detailed walkthrough of the **Query Summary** page.
 
-<div style="position: relative; padding-bottom: calc(48.68055555555556% + 41px); height: 0; width: 100%"><iframe src="https://demo.arcade.software/MTzviZtIiN7ZCmNFVJg4?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;" title="Performance - Queries - Query Summary Page Walkthrough"></iframe></div>
+<div style="position: relative; padding-bottom: calc(48.68055555555556% + 41px); height: 0; width: 100%">
+  <iframe
+    src="https://demo.arcade.software/MTzviZtIiN7ZCmNFVJg4?embed"
+    frameborder="0"
+    loading="lazy"
+    webkitallowfullscreen
+    mozallowfullscreen
+    allowfullscreen
+    style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;"
+    title="Performance - Queries - Query Summary Page Walkthrough"
+  ></iframe>
+</div>
+
+## Query Sources
 
-</br>
+A query's source is the location in your application's code that triggered the query. Sentry automatically determines the query source by examining the stack traceh and attaches the information to the query data. In order to reduce the SDK's performance impact on your application, Sentry only attaches query sources to queries that are slower than a <PlatformLink to="/configuration/options">configurable threshold</PlatformLink>. If query sources are often missing, you can reduce this threshold.
```

Comment:
```suggestion
A query's source is the location in your application's code that triggered the query. Sentry automatically determines the query source by examining the stack trace and attaches the source to the query data. To reduce the performance impact, Sentry only attaches query sources to queries that are slower than a <PlatformLink to="/configuration/options">configurable threshold</PlatformLink>. If you want to see more query sources, you can reduce this threshold.
```

Reply:
@vivianyentran I think that's because you had that particular platform selected. The link doesn't specify JavaScript